btr-web/btr-main-app/components/individual-person/control/ControlOfDirectors.vue

```diff
@@ -40,7 +40,12 @@
     >
       <slot name="typesOfControlHelp">
         <div class="flex flex-col gap-2">
-          <p>{{ $t('helpTexts.controlOfDirectors.p1') }}</p>
+          <p><b>{{ $t('helpTexts.controlOfDirectors.directControl.title') }}</b></p>
+          <p>{{ $t('helpTexts.controlOfDirectors.directControl.detail') }}</p>
+          <p><b>{{ $t('helpTexts.controlOfDirectors.indirectControl.title') }}</b></p>
+          <p>{{ $t('helpTexts.controlOfDirectors.indirectControl.detail') }}</p>
+          <p><b>{{ $t('helpTexts.controlOfDirectors.significantInfluence.title') }}</b></p>
+          <p>{{ $t('helpTexts.controlOfDirectors.significantInfluence.detail') }}</p>
```

Comment:
I think this would be a little nicer in a `v-for="controlDesc in ['directControl', 'indirectControl', 'significantInfluence']"...`

Reply:
I find it more easily readable if it is like this then in v-for.
v-for can become cumbersome and can add additional overhead on deciphering what was written. 
I think I suggested him if we have small number of repeated HTML lines it might be better to keep away from looping it.

Reply:
To me a v-for simplifies the block because you know it is the same 2 lines of code for for x things rather than needing to decipher all the lines for differences, but I am fine if it stays this way